### PR TITLE
Change/simplify landing page

### LINKED ISSUE #180: Simplify the bottom of the landing page
The current landing page is WILD!

Landing page as of 2020-12-08

It needs to provide the smoothest experience for users to get started with Predictive Text Studio

Here's what I think it should display, in ASCII art!

```
Step 1: What is the name of your language?
[  <language name dropdown with autocomplete>  ]

Step 2: Add a word list (help)
    [ Upload File | From Google Sheets URL ]

  |                                         |
  | Drag and drop an Excel .xlsx file here! |
  |                    or                   |
  |            [ Browse file... ]           |
  |                                         |
  |                                         |

[                   Continue                  ]
```

 - [x] remove the download KMP button (replace with continue button; see #165)
 - [x] split button for uploading from Excel or from a Google Sheets spreadsheet
 - [x] No more BCP-47! It should ask the language name. It will try to match with the most appropriate BCP-47 tag in the background using the Keyman API
 - [x] ~~links to an explanation of what a "word list" actually is~~ (moved to #203)

### LINKED ISSUE #197: Make it clear on the landing page that users should upload an Excel .xlsx file!
It should be explicity marked: please upload an Microsoft Excel spreadsheet (.xlsx) somewhere on the landing page

### PR DESCRIPTION
**Pull request by zprobs**
Updated the landing page to have a clearer layout. Created toggle for Google Sheets and File Upload. Changed Language input from BCP47 to dropdown.

Resolves #180 
Merged with PR #165 

Note that as of right now Cypress tests for uploading files are not cooperating.

### CODE REVIEW

File: cypress/integration/landingPage.spec.ts [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+      const event = { dataTransfer: { files: [testFile] } };
+
+      cy.get("@quick-start")
+        .data("upload-dropzone")
+        .trigger("dragenter", event);
+
+      cy.get("@quick-start").data("upload-dropzone").trigger("drop", event);
+
+      cy.data("landing-page-continue-button").click();
+      // Step3: Check if it goes to language page to verify
```
Comment (eddieantonio) [inline cypress/integration/landingPage.spec.ts:111]:
```suggestion
      // Step 3: Check if it goes to language page to verify
```

File: src/app/components/AutoComplete.svelte [hunk 2]
```diff
@@ -94,6 +97,7 @@
         index -= 1;
       }
     } else if (key === keyboardKey.Enter) {
+      e.preventDefault();
```
Comment (eddieantonio) [inline src/app/components/AutoComplete.svelte:100]: This fixes #212!

File: src/app/components/AutoComplete.svelte [hunk 3]
```diff
@@ -165,7 +174,11 @@
 <svelte:window on:keydown={handleKeydown} />
 <div class="autocomplete mb-m">
   {#if label !== ''}
-    <p class="autocomplete__label">{label}</p>
+    {#if bold}
+      <p class="autocomplete__label">{label}</p>
+    {:else}
+      <p class="autocomplete__label__normal">{label}</p>
+    {/if}
```
Comment (eddieantonio) [inline src/app/components/AutoComplete.svelte:181]: I will refactor this in a future PR, but there's a way of tidying this up. Basically, there should be a `bold` modifier, and then that modifier can be toggled with  the `bold` variable.

See: https://svelte.dev/tutorial/classes

File: src/app/components/BCP47Tag.svelte [hunk 4]
```diff
@@ -2,25 +2,36 @@
   import InputField from "./InputField.svelte";
   import * as bcp47 from "bcp-47";
   import worker from "../spawn-worker";
+  import { createEventDispatcher } from "svelte";
 
   let errorMessage: string = "";
+  const dispatch = createEventDispatcher();
 
-  function validateAndSetLanguage(event: CustomEvent) {
+  function onInputValue(event: CustomEvent) {
```
Comment (eddieantonio) [inline src/app/components/BCP47Tag.svelte:10]: Why change the name to `onInputValue()` when `validateAndSetLanguage()` is more self-documenting?

File: src/app/components/GoogleSheetsInput.svelte [hunk 5]
```diff
@@ -161,5 +161,6 @@
     fullWidth={true} />
   <button
     class="button button--primary button--shadow quick-start__submit"
+    type="button"
```
Comment (eddieantonio) [inline src/app/components/GoogleSheetsInput.svelte:164]: 👍🏼  I always forget that the default type of a `<button>` is `type="submit"`

File: src/app/components/Upload.svelte [hunk 6]
```diff
@@ -113,7 +113,7 @@
   on:dragover|preventDefault={handleDragOver}
   data-cy="upload-dropzone">
   <img role="presentation" src="icons/upload-solid.svg" alt="" />
-  <span>Drag and drop here</span>
+  <span>Drag and drop an Excel .xlsx file here!</span>
```
Comment (eddieantonio) [inline src/app/components/Upload.svelte:116]: EXCELLENT! 👍🏼

**Review by eddieantonio — APPROVED** [review]
LGTM!

There are a few comments, but they can be cleaned up in a future PR!

File: src/app/components/BCP47Tag.svelte (hunk 4 above)
Comment (zprobs) [reply src/app/components/BCP47Tag.svelte:10]: I believe this change came from @IanSeng  PR #165,

**Pull request merged**